src/pages/scripts/tutorials/script-result/index.md

```diff
@@ -1,50 +1,49 @@
 # Setting Script Result
 
-InDesign now has the functionality to "set result" of a UXP script. After execution, a script can have its own result which then can be used for debugging purposes on InDesign Server. The result of the script can also be stored in a variable and be used by another script in both InDesign App and InDesign Server. Setting the script result can be accomplished using the script.`setResult()` API, the sections below demonstrate the various use cases. 
+Since InDesign v18.4, you can "set result" of a UXP script. 
+
+After execution, a script can have its own result which then can be used for debugging purposes on InDesign Server. The result of the script can also be stored in a variable and be used by another script in both InDesign App and InDesign Server. Setting the script result can be accomplished using the `script.setResult()` API.
 
 ## Usage
 The following line sets the result of the UXP script being executed to the string `"Hello World!"`.
 
 ```js
+const script = require("uxp").script;
 script.setResult("Hello World!");
 ```
----
+
+
```

Comment:
Misleading title. The example below only talks about setting/reading the script value. The title should not be a use case, instead it should summarize the content. @anoopvalomkot

Reply:
@padmkris123 Please check if below is appropriate.

The title could be

**## Returning script result**

and the content can be below

**Scripts can return a result by setting appropriate value using setResult() API. This result is returned back to the entity that invoked this script. If this script was executed from InDesign's scripts panel, then the result value is ignored. For scripts executed in InDesign Server, the result value is returned to the client that requested for the script execution. If the script is forked by another script using app.doScript() API, the result value set by called script is returned to the caller script.**

Reply:
Thank you for providing this @anoopvalomkot 
QQ, what does 'client' mean in w.r.t InDesign Server?
Secondly, you say above that InDesign Scripts panel will ignore the result, but line 47 says: "The same functionality is also available on the InDesign App". Can you clarify

Reply:
@padmkris123 client here is a soap client that requests InDesign server to execute a UXP/Extend script (usually with arguments). The script is then executed by InDesign server and response is sent back to the client.

Reply:
**"The same functionality is also available on the InDesign App".**

This is about the doScript usecase, where one script can invoke another script. The return value from called script is relayed to the caller script in this case.

Reply:
doScript() API is documented at https://developer.adobe.com/indesign/dom/api/a/Application/
There is no direct anchor link available on that page to this API.

Reply:
JIRA ticket as requested by you, please feel free to add more information in the ticket if required.
https://jira.corp.adobe.com/browse/ID-4241896